jpf-symbc/build.gradle

```diff
@@ -0,0 +1,192 @@
+    )
+}
+
+apply from: "gradle/ide-support.gradle"
+apply from: "gradle/distribution.gradle"
+apply from: "gradle/source-sets.gradle"
+apply from: "gradle/build-resources.gradle"
+
+dependencies {
+
+    //implementation project(':jpf-core')
+    def jpfLibs = fileTree("${project(':jpf-core').buildDir}") { include '*.jar' }
+    def extLibs = fileTree(dir: 'lib', include: ['*.jar'])
+    def libs = extLibs + jpfLibs
+
+    //implementation 'org.opt4j:opt4j-core:3.1.4' //Tests are falling after implimetation
+    implementation 'org.sat4j:org.sat4j.core:2.3.1'
+    implementation 'jaxen:jaxen:1.1'
+    implementation 'redis.clients:jedis:2.0.0'
+    implementation 'com.github.green-solver:green:0.2.0'
+    implementation 'commons-math:commons-math:1.2'
+    implementation 'commons-lang:commons-lang:2.4'
+    implementation 'org.choco-solver:choco-solver:4.10.4'
+    implementation 'bcel:bcel:5.1'
+    implementation 'com.googlecode.aima-java:aima-core:0.10.5'
+    implementation 'com.martiansoftware:jsap:2.1'
+    
+    implementation libs
+    classesImplementation libs
+    peersImplementation libs
+    examplesImplementation libs
+    testImplementation libs
+
+    testImplementation "junit:junit:4.12"
+    
+}
+
+task compile {
+    group = "SPF Build"
+    description = "Compiles all JPF symbc sources."
+
+    // These are automatic generated tasks from the Java Gradle Plugin.
+    // Gradle is able to infer the order of the source sets
+    // due to the compileClasspath attribute
+    dependsOn compileExamplesJava
+    dependsOn compileTestJava
+}
+
+task createJpfClassesJar(type: Jar) {
+    archiveName = "jpf-symbc-classes.jar"
+    destinationDir = file("${buildDir}")
+
+    group = "SPF Jars"
+    description = "Creates the ${archiveName} file."
+
+    dependsOn compile
+    dependsOn copyResources
+
+    from sourceSets.main.java.outputDir
+    from sourceSets.annotations.java.outputDir
+    from sourceSets.classes.java.outputDir
+}
+
+task createJpfJar(type: Jar) {
+    archiveName = "jpf-symbc.jar"
+    destinationDir = file("${buildDir}")
+
+    group = "SPF Jars"
+    description = "Creates the ${archiveName} file."
+
+    dependsOn compile
+    dependsOn copyResources
+
+    from sourceSets.main.java.outputDir
+    from sourceSets.peers.java.outputDir
+    from sourceSets.annotations.java.outputDir
+    from sourceSets.classes.java.outputDir
+
+    manifest {
+        attributes "Implementation-Title": "Symbolic Pathfinder system"
+        from manifestCommonAttrbutes
+    }
+}
+
+task createAnnotationsJar(type: Jar) {
+    archiveName = "jpf-symbc-annotations.jar"
+    destinationDir = file("${buildDir}")
+
+    group = "SPF Jars"
+    description = "Creates the ${archiveName} file."
+
+    dependsOn compile
+    dependsOn copyResources
+
+    from sourceSets.annotations.java.outputDir
+}
+
+task buildJars {
+    group = "SPF Build"
+    description = "Generates all JPF symbc jar files."
```

Comment:
change to "SPF jar files" or "jpf-symbc jar files"

Reply:
Done